cypress/support/commands.js

```diff
@@ -48,10 +48,62 @@ Cypress.Commands.add(
   }
 );
 
-Cypress.Commands.add('getEditor', () => cy.get('[data-slate-editor=true]'));
+Cypress.Commands.add('getEditor', () => cy.get('[data-testid="note-editor"]'));
 
 // This is not a great way to target elements, but
-// toastify doesn't support adding data attributes
-Cypress.Commands.add('selectToastByContent', (content) =>
+// Toastify doesn't support adding data attributes
+Cypress.Commands.add('getToastByContent', (content) =>
   cy.get('.Toastify__toast-body').should('be.visible').contains(content)
 );
+
+Cypress.Commands.add('getSidebarNoteLink', (noteTitle) => {
+  cy.get(`[data-testid="sidebar-note-link-${noteTitle}"]`);
+});
+
+Cypress.Commands.add('getNoteLinkElement', (noteTitle) => {
+  cy.get(`[data-testid="note-link-element-${noteTitle}"]`);
+});
+
+// Gets the element where the note's title is editable
+// contains() by default allows for partial matches
+// But this RegExp finds the exact match of noteTitle
+// '^' asserts position at start of the string
+// '$' asserts position at end of string
+Cypress.Commands.add('getNoteTitle', (noteTitle) => {
+  cy.get('[data-testid="note-title"]').contains(
+    new RegExp('^' + noteTitle + '$')
+  );
+});
+
+// Gets a linked or unlinked reference by the note's title
+// You must pass "linked" or "unlinked" for referenceType
+// Linked references have the testid of "linked-reference"
+// Unlinked references have the testid of "unlinked-reference"
+Cypress.Commands.add('getReference', (noteTitle, referenceType) => {
+  cy.get(`[data-testid="${referenceType}-reference"]`).contains(
+    new RegExp('^' + noteTitle + '$')
+  );
+});
+
+// Asserts the amount of references of a specific type a note should have
+// You must pass "linked" or "unlinked" for referenceType
+Cypress.Commands.add(
+  'numberOfReferencesShouldEqual',
```

Comment:
What do you think about checking the backlink branch title instead (the one that says "X Linked References" or "X Unlinked References"? This will also test that the number displayed is accurate.

This changes the logic such that it'll now be checking the number of references (instead of the number of notes that include a reference), so you'll need to update the tests to account for this. I think this new approach better fits the name of this command, though.